recipes/re2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "re2" %}
+{% set version = "2016.11.01" %}
+{% set dashversion = version.replace('.', '-') %}
+{% set sha256 = "01ee949f03e1c4057dc533cf139f967fb1b427015769d53b9ee07757631e9669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/google/{{ name }}/archive/{{ dashversion }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  #features:
+  #  - vc9     # [win and py27]
+  #  - vc10    # [win and py34]
+  #  - vc14    # [win and py35]
+  detect_binary_files_with_prefix: True
+  #msvc_compiler: 14.0
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.1
```

Comment:
FYI, CMake >= 3.4 is needed for this feature, https://github.com/google/re2/blob/master/CMakeLists.txt#L23